src/patch.js

```diff
@@ -22,6 +22,33 @@ var walker = require('./walker'),
     getWalker = walker.getWalker,
     setWalker = walker.setWalker;
 
+/**
+ * @const {boolean}
+ */
+var IS_PRODUCTION = process.env.NODE_ENV === 'production';
+
+if (!IS_PRODUCTION) {
+  var assertNoUnclosedTags = function(root) {
+    var openElement = getWalker().getCurrentParent();
+    if (!openElement) {
+      return;
+    }
+
+    var openElements = [];
+    while(openElement && openElement !== root) {
+      openElements.push(openElement);
```

Comment:
I was thinking about using something like https://www.npmjs.com/package/ancestors to get the ancestors (though it would need to support an optional root node), but don't think it would get stripped in a production build. Any thoughts?